postcss.config.js

```diff
@@ -10,6 +10,13 @@ module.exports = {
 				outlines: '#007cba',
 			},
 			themes: {
+				'woocommerce-profile-wizard__body': {
```

Comment:
Will these color values need to be updated?  If so, I can log some issues around them.  Also curious why the `woocommerce-page` values are all set to `$woocommerce-500`.

I expected those values to match up with [these](https://github.com/woocommerce/woocommerce/blob/master/assets/css/_variables.scss#L11) but I might be misunderstanding the purpose of these variables.

Reply:
I believe these will need to be updated, but they don't seem to be applying correctly for me in all circumstances. The primary/button colors are working (which is what we need in this PR), but changing `toggle` here doesn't seem to work. I'll log an issue for follow-up.

The link you provided are existing core colors, but are not part of the rebrand colors (with the pinks and purples).

Here are the toggle, outlines, primary, and button classes being shown:

<img width="510" alt="Screen Shot 2019-05-28 at 9 04 00 AM" src="https://user-images.githubusercontent.com/689165/58480096-941ac200-8127-11e9-8804-cf013c2e6779.png">

Reply:
https://github.com/woocommerce/woocommerce-admin/issues/2322